src/components/DemoPage/DemoPage.tsx

```diff
@@ -56,9 +60,47 @@ const formConfig = {
   },
 };
 
+const initLogData = [
```

Comment:
It would be better to create some timeout to display the demo log line by line, so that it would be more similar to a real live log on buildDetails page.